collections/high_availability/roles/drbd/tasks/main.yml

```diff
@@ -36,67 +36,140 @@
   changed_when: False
+  ignore_errors: true
 
-- name: command <|> Create needed resources
-  ansible.builtin.command: "drbdadm create-md {{ item.name }} "
+- name: filesystem █ Create a new primary partition
+  community.general.parted:
+    device: "{{ item.partitionning.device | default(omit) }}"
+    number: "{{ item.partitionning.number | default(omit) }}"
+    label: "{{ item.partitionning.label | default(omit) }}"
+    state: "{{ item.partitionning.state | default(omit) }}"
+    align: "{{ item.partitionning.align | default(omit) }}"
+    part_type: "{{ item.partitionning.part_type| default(omit) }}"
+    part_start: "{{ item.partitionning.part_start | default(omit) }}"
+    part_end: "{{ item.partitionning.part_end | default(omit) }}"
+  loop: "{{ drbd_resources }}"
+  when: item.partitionning is defined and item.partitionning is not none
+
+- name: filesystem █ Create a volume group
+  community.general.lvg:
+    vg: "{{ item.logical_volumes.vg | default(omit) }}"
+    pvs: "{{ item.logical_volumes.partition | default(omit) }}"
+  loop: "{{ drbd_resources }}"
+  when: item.logical_volumes is defined and item.logical_volumes is not none
+
+- name: filesystem █ Create logical volume
+  community.general.lvol:
+    vg: "{{ item.logical_volumes.vg | default(omit) }}"
+    lv: "{{ item.logical_volumes.lv | default(omit) }}"
+    size: "{{ item.logical_volumes.size | default(omit) }}"
+    pvs: "{{ item.logical_volumes.partition | default(omit) }}"
+    state: "{{ item.logical_volumes.state | default(omit) }}"
+  loop: "{{ drbd_resources }}"
+  when: item.logical_volumes is defined and item.logical_volumes is not none
+
+- name: command █ Create needed resources
+  ansible.builtin.command: "/usr/sbin/drbdadm create-md {{ item.name }} --force"
   loop: "{{ drbd_resources }}"
   when:
     - (drbd_created.stdout_lines | select("match", ".*"+item.disk+"$") | list | length) < 1
 
-- name: command <|> Check if resources are up
-  ansible.builtin.command: 'drbdadm status'
+- name: command █ Check if resources are up
+  ansible.builtin.command: "/usr/sbin/drbdadm status"
   register: drbdadm_status
   changed_when: False
 
-- name: command <|> Up needed resources
-  ansible.builtin.command: "drbdadm up {{ item.name }} "
+- name: command █ Up needed resources
+  ansible.builtin.command: "/usr/sbin/drbdadm up {{ item.name }}"
   loop: "{{ drbd_resources }}"
   when:
     - (drbdadm_status.stdout_lines | select("match", "^"+item.name+".*") | list | length) < 1
 
-- name: pause <|> Wait for resources to be up and used
-  ansible.builtin.pause:
-    seconds: "{{ drbd_pause_seconds }}"
-
 - name: Format FS if needed
   run_once: true
   block:
 
-    - name: command <|> Check if resources are not already used
-      ansible.builtin.shell: "drbdadm status | sed -n '/^{{ item.name }}/,/^$/{p;/^$/q}'"
+    - name: command █ Check if resources are not already used
+      ansible.builtin.shell: "/usr/sbin/drbdadm status | grep {{ item.name }}"
```

Comment:
Lint is reporting this issue:
```
risky-shell-pipe

Shells that use pipes should set the pipefail option.

```

Can you check this ?

Reply:
Done.